tests/unit/mocked/replicator_test.py

```diff
@@ -127,34 +142,52 @@ def test_create_replication_errors(self):
             )
 
     def test_list_replications(self):
-        with mock.patch('cloudant.database.CloudantDatabase.all_docs') as mock_all_docs:
+        """ test retrieve replications"""
+        def getitem(name):
+            db_dict = {'_replicator': CouchDatabase(
+                mock_account,
+                '_replicator'
+            )}
+            return db_dict[name]
+        with mock.patch('cloudant.database.CouchDatabase.all_docs') as mock_all_docs:
             mock_all_docs.return_value = {
                 "rows": [
-                    {"doc":"replication_1"},
-                    {"doc": "replication_2"}
+                    {"id": "replication_1", "doc": {"_id": "replication_1"}},
+                    {"id": "replication_2", "doc": {"_id": "replication_2"}}
                 ]
             }
             mock_account = mock.Mock()
-            repl = ReplicatorDatabase(mock_account)
+            mock_account.__getitem__ = mock.Mock()
+            mock_account.__getitem__.side_effect = getitem
+            repl = Replicator(mock_account)
 
             self.assertEqual(
                 repl.list_replications(),
-                ['replication_1', 'replication_2']
+                [{"_id": "replication_1"}, {"_id": "replication_2"}]
             )
+
     def test_replication_state(self):
         """test replication state method"""
+        def getitem(name):
```

Comment:
I moved `mock_account` to the setUp since it is used in each test.  I also removed the need for the `getitem` function by replacing the `mock_account.side_effect` with `mock_account.return_value`.  Done in 9507fed.